### ISSUE TITLE
Says it runs on OSX 10.11+ but isn't allowed to run unless 10.12+

### ISSUE THREAD

**Issue by ksaj** [portacle/portacle#121]

**Comment by ksaj** [comment]
Versions of OSX prior to OSX 10.12 show the icon crossed out and a warning that the application can only run on 10.12.  Are there any archives with an older version that *will* run under 10.11.x?

**Comment by nazarsh** [comment]
@ksaj Did you check the https://github.com/portacle/portacle/releases to see if any of them would work for you?

**Comment by ksaj** [comment]
There is only one Mac release, and its the one that doesn't work under 10.11.x (although the docs say it does).  Anyway, I just installed emacs and configured it with everything I need, so it's all good for me. At least I did get portacle to install on a Linux box and after deciding I liked it, basically built it on the Mac from scratch.

**Comment by Shinmera** [comment]
What are you talking about? Most of the releases include Mac downloads.

**Comment by ksaj** [comment]
You are aggressively ignoring what I've already said quite clearly. I'm running 10.11.6 and when I download the Portacle image for Mac, which the info says should work, the app installer is shown with a cross-out (like the no smoking sign) because it can't be installed on this version of OSX. That's what I'm talking about (and now simply repeating). 

I won't be attempting to remedy it since I already have installed emacs etc manually and it is working fine. Which is also why I closed the thread, and am closing it again. I'm simply no longer interested in Portacle. Why reopen it when I also already stated that, too?

I will not be responding if this gets reopened yet again. It is no longer of interest to me since I will not be using Portacle.
